dc.sh

```diff
@@ -52,9 +52,11 @@ function publish() {
    source ./service/docker/bootstrap.env
     export FULL_PATH=$(realpath $PATH_TO_CONTRACTS)
 
+ echo $SIGNING_PRIVATE_KEY
+ echo $ENCRYPTION_PRIVATE_KEY
     if [[ -d "$FULL_PATH" ]]; then
         pushd $PATH_TO_CONTRACTS > /dev/null
-        ./gradlew p8eClean p8eBootstrap --info
+        ./gradlew p8eClean p8eCheck p8eBootstrap --info && ./gradlew publishToMavenLocal -xsignMavenPublication --info
```

Comment:
Why use `&&` instead of having all the tasks in the same `./gradlew` execution?

Reply:
This is more of a mechanism to force the publish task to happen after bootstrapping fully. I found combining them under one gradlew command that the built ContractHash / ProtoHash would not exist in the published artifact. In theory it shouldnt be a problem but it appears anyway - also one of the first things pierce asked me when I originally had issues with it.